### PR TITLE
refactor(dependencies): removed vulnerabilities by upgrading packages

### PR DESCRIPTION
**Pull request by dazlious**
Hope to get rid of those:
https://github.com/titel-media/react-apollo-multiple-clients/network/alerts

### CODE REVIEW

File: src/provider.test.jsx [hunk 1]
```diff
@@ -30,20 +28,17 @@ const clientList = {
 };
 
 describe('React Apollo Multiple Clients', () => {
-  /* eslint-disable no-console */
   beforeAll(() => {
     global.consoleError = console.error;
-    console.error = () => {};
+    console.error = () => null;
```
Comment (majoravery) [inline src/provider.test.jsx:33]: Why is this changed to null?

File: .eslintrc.js [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
-    'jest/prefer-to-be-undefined': 'error',
-    'max-len': ['error', { code: 120, ignoreStrings: true, ignoreTemplateLiterals: true }],
-    'no-unused-expressions': ['error', { allowTernary: true }],    
-    'react/no-array-index-key': 'off',
-    'curly': ['error', 'all'],
+    'react/display-name': 'off', // TODO: fix once hs-lint supports react config
   },
+  overrides: [
+    {
+      files: ['**/*.test.js*', '**/*.spec.js*'],
```
Comment (majoravery) [inline .eslintrc.js:17]: How come we're overriding the rules for test files?

Comment (dazlious) [reply .eslintrc.js:17]: For tests we do not need to be that strict. For instance console.log and other stuff can be used there. In our production code users might see stuff. In general I agree, but this will be solved as soon as hs-lint introduces linting rules for react ;)

File: src/provider.test.jsx (hunk 1 above)
Comment (dazlious) [reply src/provider.test.jsx:33]: This is just to prevent the error to be shown while test suite is running. I could have done this explicitly for the specific tests instead of "all" ;)

File: package.json [hunk 3]
```diff
@@ -21,30 +21,32 @@
     "test": "jest --coverage ./src"
   },
   "dependencies": {
-    "prop-types": "^15.6.2",
-    "react": "^16.7.0"
+    "apollo-client": "^2.6.3",
+    "graphql": "^14.4.2",
```
Comment (jortizsao) [inline package.json:25]: didn't we have the `graphql` dependency nor `apollo-client` before?

Comment (dazlious) [reply package.json:25]: Nope. But that is not important. I just wanted to get rid of the peer dependency warnings
